src/solvers/umfpack.jl

```diff
@@ -234,9 +237,14 @@ A shallow copy of UmfpackLU to use in multithreaded applications. This function
 It can also take transposed or adjoint `UmfpackLU`s.
 """
 # Not using simlar helps if the actual needed size has changed as it would need to be resized again
-Base.copy(F::UmfpackLU, ws=UmfpackWS(F)) = UmfpackLU(
-    F.symbolic,
-    F.numeric,
+Base.copy(F::UmfpackLU, ws=UmfpackWS(F); copypointers = false) = 
+    finalizer(umfpack_free_symbolic_nl,
+    UmfpackLU(
+    # symbolic and numeric are writeable, so must be copied.
+    # This may be disadvantageous when we're solving. 
+    # In that case both are only read from as far as I know.
+    copypointers ? F.symbolic : C_NULL, 
```

Comment:
Is there an umfpack method that copies these?

Reply:
Not that I'm aware of. Just saving to file and back. I can do that, but it's cumbersome.

Reply:
@Keno Is there an in-memory `Libc.FILE`? I could pass that to `save` and `load`. We'd be serializing essentially but it's the only method of copying.

Reply:
Nevermind it takes a filename, so I'll just use `mktemp`

Reply:
It would be surprising for copy to do file io. Let's just leave it as is and make a feature request for an in memory copy